express/validations/teams.js

```diff
@@ -17,6 +17,15 @@ const validations = {
 				),
 			filterValue: Joi.string()
 				.regex(common.filterValueRegex),
+			pageNumber: Joi.number()
+				.integer()
+				.positive(),
```

Comment:
You can add default value here so you'll always have 1 page as default, same goes to instancesNumber